Shared/Store/BaseAccountStore.swift

```diff
@@ -0,0 +1,64 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+import RxSwift
+import RxCocoa
+import FxAClient
+import SwiftKeychainWrapper
+import WebKit
+
+class BaseAccountStore {
+    internal var localKeychainWrapper: KeychainWrapper
+    internal var sharedKeychainWrapper: KeychainWrapper
```

Comment:
I don't understand the need for two keychains here.

I'd also *strongly* advise you to convince yourself that you are using the keychain as the one used in the `lockbox-fxa-sync-ios` library. 

This might be as simple as using the `KeychainWrapper.sharedAppContainerKeychain` everywhere we're using `KeychainWrapper.standard`.